server/src/test/java/org/elasticsearch/health/components/controller/LackOfQourumTests.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.health.components.controller;
+
+import org.elasticsearch.Version;
+import org.elasticsearch.cluster.ClusterName;
+import org.elasticsearch.cluster.ClusterState;
+import org.elasticsearch.cluster.coordination.CoordinationMetadata;
+import org.elasticsearch.cluster.metadata.Metadata;
+import org.elasticsearch.cluster.node.DiscoveryNode;
+import org.elasticsearch.cluster.node.DiscoveryNodes;
+import org.elasticsearch.common.transport.TransportAddress;
+import org.elasticsearch.health.HealthIndicatorResult;
+import org.elasticsearch.health.HealthStatus;
+import org.elasticsearch.test.ESTestCase;
+
+import java.util.Set;
+
+import static java.util.Collections.emptyMap;
+import static org.elasticsearch.cluster.node.DiscoveryNodeRole.DATA_ROLE;
+import static org.elasticsearch.cluster.node.DiscoveryNodeRole.INGEST_ROLE;
+import static org.elasticsearch.cluster.node.DiscoveryNodeRole.MASTER_ROLE;
+import static org.elasticsearch.cluster.node.DiscoveryNodeRole.TRANSFORM_ROLE;
+
+public class LackOfQourumTests extends ESTestCase {
+
+    public void testIsGreenIfThereIsOnlyOneNode() {
+        ClusterState clusterState = ClusterState.builder(new ClusterName("test-cluster"))
+            .nodes(
+                DiscoveryNodes.builder()
+                    .add(new DiscoveryNode("node1", transportAddress(), emptyMap(), Set.of(MASTER_ROLE), Version.CURRENT))
+                    .build()
+            )
+            .metadata(
+                Metadata.builder()
+                    .coordinationMetadata(
+                        CoordinationMetadata.builder()
+                            .lastCommittedConfiguration(new CoordinationMetadata.VotingConfiguration(Set.of("node1")))
+                            .build()
+                    )
+                    .build()
+            )
+            .build();
+
+        HealthIndicatorResult noEligibleMasterNodes = ClusterCoordination.lackOfQuorum(clusterState);
+
+        assertEquals("lack-of-quorum", noEligibleMasterNodes.name());
+        assertEquals(HealthStatus.GREEN, noEligibleMasterNodes.status());
+        assertEquals("Instance can form a quorum", noEligibleMasterNodes.summary());
```

Comment:
Here and in other tests:

```suggestion
        assertThat(noEligibleMasterNodes, equalsTo(HealthIndicatorResult.of("lack-of-quorum", "component", "Instance can form a quorum", GREEN)))
```

This way all fields are asserted at once and it might be easier to spot the difference between entire expected and actual object